modules/@apostrophecms/widget-type/index.js

```diff
@@ -161,6 +161,14 @@ module.exports = {
           ...self.getWidgetsBundles(`${widget.type}-widget`)
         };
 
+        if (widget.aposPlaceholder === true) {
+          self.schema.forEach(field => {
+            if (field.placeholder !== undefined) {
+              widget[field.name] = field.placeholder;
+            }
+          });
+        }
```

Comment:
I saw no need to shallow clone the widget so it doesn't save the placeholder values in the database